src/date/fromString.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Returns a Date from string value format espected format dd[/]mm[/]yyyy
+ * @param {string} dateFrom
+ * @return Date
+ */
+
+import { DateTime } from "luxon";
+import { nullValue } from "../generator/out/not_implemented";
```

Comment:
`import nullValue` is unnecessary, it is not used.

Reply:
Fixed at commit d440aae2a58e17de941b6a9953099207128944ec

Reply:
We need to solve the string representation format for this date values in a model preferences dependant way. 
Issue #33 was created.